inputs/Glider.php

```diff
@@ -5,6 +5,11 @@
 use GameOfLife\Field;
 use Ulrichsg\Getopt;
 
+/**
+ * Class Glider
```

Comment:
Please remove unnecessary „Class Glider“. This helps no one. Every developer sees that this is a class with the name „Glider“. Please just remove this line. The next sentence is a perfect summary documentation of the class and can be used as first line.